package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "db-migrate-mysql",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
Remove this please, you never should ever alter the package.json, unless you install a new package, which also would need to be justified.